packages/create-catalyst/src/utils/get-latest-core-tag.ts

```diff
@@ -0,0 +1,19 @@
+import * as z from 'zod';
+
+export const getLatestCoreTag = async () => {
+  const latestReleasesUrl = 'https://api.github.com/repos/bigcommerce/catalyst/releases';
+
+  const response = await fetch(latestReleasesUrl);
+
+  const releases = z.array(z.object({ tag_name: z.string() })).parse(await response.json());
+
+  const latestCoreRelease = releases.find((release) =>
+    release.tag_name.startsWith('@bigcommerce/catalyst-core'),
+  );
```

Comment:
Is this relying on the releases being ordered from most recent to least recent? Does the API guarantee this behavior (a quick look at their docs didn't seem to say that it did, but I may be missing something)

Reply:
You're correct - unfortunately, I couldn't find anything to guarantee the order will always start with most recent 😞 

I'm open to suggestions for alternatives... I could modify the request to add a maximum `per_page` param and sort the response myself, but this still doesn't provide a 100% guarantee. I'm also trying to avoid any rate limits from GitHub ([60 requests per hour](https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api?apiVersion=2022-11-28#primary-rate-limit-for-unauthenticated-users)), otherwise I'd be open to paginating the response to grab every release and sort in-memory (cons: inefficient, and could cause issues as the list grows in the future)

Reply:
Refactored this PR to pull the latest core tag by parsing the [raw content of the `apps/core/package.json` file](https://raw.githubusercontent.com/bigcommerce/catalyst/main/apps/core/package.json) on `main` 👍